tests/data/l7irish/data.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import hashlib
+import os
+import random
+import shutil
+
+import numpy as np
+import rasterio
+from rasterio.crs import CRS
+
+SIZE = 64
+
+np.random.seed(0)
+random.seed(0)
+
+wkt = """
```

Comment:
This can be made simpler:
```pycon
>>> import rasterio
>>> src = rasterio.open("L71226098_09820011112_B10.TIF")
>>> src.crs
CRS.from_epsg(32719)
```
So you can replace this giant string with that one line on line 53.